Fix find skill author

forslund opened the pull request:
This attempts to fix #1523, the function to get skill author seem to only work when the skill is first added to the .gitmodules, this uses the .gitmodules-file instead of relying on the content in the diff.

devops-mycroft commented:
Voight Kampff Integration Test Failed ([Results](https://reports.mycroft.ai/skills/PR-1524)). 
Mycroft logs are also available: [skills.log](https://reports.mycroft.ai/skills/PR-1524/logs/skills.log), [audio.log](https://reports.mycroft.ai/skills/PR-1524/logs/audio.log), [voice.log](https://reports.mycroft.ai/skills/PR-1524/logs/voice.log), [bus.log](https://reports.mycroft.ai/skills/PR-1524/logs/bus.log), [enclosure.log](https://reports.mycroft.ai/skills/PR-1524/logs/enclosure.log)

devops-mycroft commented:
Voight Kampff Integration Test Failed ([Results](https://reports.mycroft.ai/skills/PR-1524)). 
Mycroft logs are also available: [skills.log](https://reports.mycroft.ai/skills/PR-1524/logs/skills.log), [audio.log](https://reports.mycroft.ai/skills/PR-1524/logs/audio.log), [voice.log](https://reports.mycroft.ai/skills/PR-1524/logs/voice.log), [bus.log](https://reports.mycroft.ai/skills/PR-1524/logs/bus.log), [enclosure.log](https://reports.mycroft.ai/skills/PR-1524/logs/enclosure.log)

devops-mycroft commented:
Voight Kampff Integration Test Failed ([Results](https://reports.mycroft.ai/skills/PR-1524)). 
Mycroft logs are also available: [skills.log](https://reports.mycroft.ai/skills/PR-1524/logs/skills.log), [audio.log](https://reports.mycroft.ai/skills/PR-1524/logs/audio.log), [voice.log](https://reports.mycroft.ai/skills/PR-1524/logs/voice.log), [bus.log](https://reports.mycroft.ai/skills/PR-1524/logs/bus.log), [enclosure.log](https://reports.mycroft.ai/skills/PR-1524/logs/enclosure.log)

devops-mycroft commented:
Voight Kampff Integration Test Failed ([Results](https://reports.mycroft.ai/skills/PR-1524)). 
Mycroft logs are also available: [skills.log](https://reports.mycroft.ai/skills/PR-1524/logs/skills.log), [audio.log](https://reports.mycroft.ai/skills/PR-1524/logs/audio.log), [voice.log](https://reports.mycroft.ai/skills/PR-1524/logs/voice.log), [bus.log](https://reports.mycroft.ai/skills/PR-1524/logs/bus.log), [enclosure.log](https://reports.mycroft.ai/skills/PR-1524/logs/enclosure.log)

forslund commented:
See https://hudson.mycroft.team/blue/organizations/jenkins/mycroft-skills/detail/PR-1524/3/pipeline/ for a run where game-zork was found and tests ran as expected.

I rebased it since and removed the game-zork commit.

krisgesling reviewed (approved):
Working great, thanks again!
